recipe/migrations/libopenvino_dev202330.yaml

```diff
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for libopenvino-dev 2023.3.0
+  kind: version
+  migration_number: 1
+migrator_ts: 1705841975.2201185
+libopenvino-dev:
```

Comment:
- not sure what is proper name here `libopenvino_dev` or `libopenvino-dev` (actual package name)
- conda bot has not created automatically migration file for OpenVINO, do you know why?

Reply:
This needs to be an underscore. `-` are not allowed for the key here.

Reply:
thanks @xhochy, fixed
do you know answer on the second question?

Reply:
No.